File: problems/binary-search/boilerplate-full/function.js
##USER_CODE_HERE##

const input = require('fs').readFileSync('/dev/problems/binary-search/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
const size_nums = parseInt(input.shift());
const nums = input.splice(0, size_nums).map(Number);
  const target = parseInt(input.shift());
const result = search(nums, target);
console.log(result);
    

File: problems/binary-search/boilerplate-full/function.rs
use std::fs::read_to_string;
use std::io::{self};
use std::str::Lines;

##USER_CODE_HERE##

fn main() -> io::Result<()> {
  let input = read_to_string("/dev/problems/binary-search/tests/inputs/##INPUT_FILE_INDEX##.txt")?;
  let mut lines = input.lines();
  let size_nums: usize = lines.next().unwrap().parse().unwrap();
let nums: Vec<i32> = lines.next().unwrap().split_whitespace().filter_map(|x| x.parse().ok()).collect();
  let target: i32 = lines.next().unwrap().parse().unwrap();
  let result = search(nums, target);
  println!("{}", result);
  Ok(())
}
